mlflow/shap.py

```diff
@@ -239,6 +240,12 @@ def predict_function(X) -> pred:
     import matplotlib.pyplot as plt
     import shap
 
+    original_config_dict = mlflow.utils.autologging_utils.AUTOLOGGING_INTEGRATIONS
```

Comment:
Try to add  a util method in autologging_utils that would return only the list of actively enabled autologging integrations.